theme.js

```diff
@@ -203,7 +203,6 @@ export default ({
     'gradient-to-tl':
       'linear-gradient(to top left, var(--gradient-color-stops))',
   },
-  gradientColorStops: colors,
```

Comment:
Sorry for the second totally unrelated change, but IDE warned about duplicated key/value and that looks to be a correct warning. 

https://github.com/lukejacksonn/oceanwind/blob/ee88610ca1c15fcd383ecbf932c1d4db767497aa/theme.js#L190-L206

Reply:
You are totally right 🙌 don't apologise, this is all great work!